packages/terra-table/src/Table.module.scss

```diff
@@ -20,17 +20,33 @@
     th,
```

Comment:
I will remove `th` - it's unnecessary because `th` now has it's own block below

Reply:
https://github.com/cerner/terra-core/pull/2133/commits/2f7904dbe874f1087ee6fda4364abd37584a0dba